package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "react-signature-pad",
-  "version": "0.0.5",
+  "name": "react-signature-pad-component",
```

Comment:
Sorry! I didn't know I shouldn't change version and name. I send a merge request. So you can add onClear event in the original module